src/core/Svg.tsx

```diff
@@ -48,12 +48,11 @@ export const Svg: ForwardRefComponent<SvgProps, Object3D> = /* @__PURE__ */ forw
                 path.userData?.style.fill !== undefined &&
                 path.userData.style.fill !== 'none' &&
                 SVGLoader.createShapes(path).map((shape, s) => (
-                  <mesh key={s} {...fillMeshProps}>
+                  <mesh key={s} {...fillMeshProps} renderOrder={s}>
                     <shapeGeometry args={[shape]} />
                     <meshBasicMaterial
                       color={path.userData!.style.fill}
                       opacity={path.userData!.style.fillOpacity}
-                      transparent={true}
```

Comment:
Why remove `transparent`? It does affect grouping during sorting, but is that any relevant here?

Reply:
From my experiments this was the only way i found to avoid the render order issues.

Reply:
I'll investigate merging the geometries here. Definitely an issue with sorting, not z-fighting exactly.

Reply:
I think the reason this PR doesn't work with `transparent` is because the `renderOrder` should be applied based on all the meshes, not in the subgroups.

Here is a working example with transparent enabled: [sandbox](https://codesandbox.io/p/sandbox/svg-bug-gvjv2z?file=%2Fsrc%2FSvg.jsx%3A47%2C3-47%2C23)

If we update `renderOrder` it should be aligned again with the threejs example.

The introduction of `renderOrder` happened in this PR: https://github.com/mrdoob/three.js/issues/26110